sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerIdentityServiceClient.cs

```diff
@@ -17,7 +17,8 @@ namespace Azure.Security.ConfidentialLedger
     public partial class ConfidentialLedgerIdentityServiceClient
     {
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
-        public virtual HttpPipeline Pipeline { get; }
+        public virtual HttpPipeline Pipeline { get => _pipeline; }
```

Comment:
This requires a change to https://github.com/Azure/azure-sdk-for-net/blob/13e36f97bb2aa23b9c9c60cea890dbc18a1eb5b5/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerIdentityServiceClient.cs#L35

Reply:
Thanks, Chris.  There were a handful of the same issue across some other libraries as well, which I went ahead and fixed up.